packages/@o3r/workspace/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,57 @@
+import { chain, externalSchematic, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { lastValueFrom } from 'rxjs';
+import type { PackageJson } from 'type-fest';
+import { AddDevInstall, isPackageInstalled, registerPackageCollectionSchematics } from '@o3r/schematics';
+import { NgAddSchematicsSchema } from './schema';
+import { RepositoryInitializerTask } from '@angular-devkit/schematics/tasks';
+import { prepareProject } from './project-setup/index';
+
+/**
+ * Add Otter library to an Angular Project
+ * @param options
+ */
+export function ngAdd(options: NgAddSchematicsSchema): Rule {
+  return async (tree: Tree, context: SchematicContext) => {
+    const ownPackageJsonContent = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', '..', 'package.json'), {encoding: 'utf-8'})) as PackageJson;
```

Comment:
isn't `o3rWorkspacePackageJsonContent` a better name?

Reply:
same as `o3rWorkspacePackageJson` is actually its own package.json :D